src/app.js

```diff
@@ -27,3 +27,7 @@ ReactDOM.render(
   </Provider>,
   document.getElementById('app')
 );
+
+window.Intercom("boot", {
```

Comment:
It will queue the request for when `intercom.js` is downloaded. See comment about script tax. We can move this to a better location when we make it aware of the current user